<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixerDetails</class>
 <widget class="QWidget" name="MixerDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>183</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="horizontalSpacing">
      <number>11</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="3">
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
       <property name="spacing">
        <number>11</number>
       </property>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="toolTip">
          <string>MIDI Out controllers</string>
         </property>
         <property name="title">
          <string>MIDI</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="2" column="0">
             <widget class="QLabel" name="labelChorus">
              <property name="text">
               <string>Chorus:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="reverbSlider">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>MIDI OUT reverb</string>
              </property>
              <property name="maximum">
               <number>127</number>
              </property>
              <property name="pageStep">
               <number>16</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="chorusSlider">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>MIDI OUT chorus</string>
              </property>
              <property name="maximum">
               <number>127</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>16</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelPort">
                <property name="text">
                 <string>Port:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="portSpinBox">
                <property name="toolTip">
                 <string>MIDI port number</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="labelChannel_2">
                <property name="text">
                 <string>Channel:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QSpinBox" name="channelSpinBox">
                <property name="toolTip">
                 <string>MIDI channel mumber</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelReverb">
              <property name="text">
               <string>Reverb:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="reverbSpinBox">
              <property name="toolTip">
               <string>MIDI OUT reverb</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>127.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="chorusSpinBox">
              <property name="toolTip">
               <string>MIDI OUT chorus</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>127.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>11</number>
       </property>
       <item row="5" column="2">
        <widget class="QDoubleSpinBox" name="panSpinBox">
         <property name="toolTip">
          <string>Pan</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>127.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="panSlider">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Pan</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="volumeSpinBox">
         <property name="toolTip">
          <string>Volume</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>127.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="patchCombo">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Patch voice</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="drumkitCheck">
         <property name="text">
          <string>Drumkit</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelPatch">
         <property name="text">
          <string>Patch:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="channelLabel">
         <property name="toolTip">
          <string>Channel name</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelChannel">
         <property name="text">
          <string>Channel:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="partNameLineEdit">
         <property name="toolTip">
          <string>Part name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Awl::ColorLabel" name="trackColorLabel" native="true">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Track color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="volumeSlider">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Volume</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="maximum">
          <number>128</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelVolume">
         <property name="text">
          <string>Volume:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelPan">
         <property name="text">
          <string>Pan:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Awl::ColorLabel</class>
   <extends>QWidget</extends>
   <header>awl/colorlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
